main.py

```diff
@@ -146,23 +58,64 @@ def new_task():
             return_to_menu()
 
 
+def task_edition():
+    task_number = input("Podaj numer ID zadania do edycji: ")
+    if task_number.isdecimal() is False:
```

Comment:
TUtaj można lepiej sprawdzać. isdecimal zwraca od razu Treu lub False. Zatem zamiast sprawdzać czy False is False to można
```python
if not task_number.isdecimal()
```
a gdyby miałobyć True to:
```python
if task_number.isdecimal()
```

Reply:
OKI zaimplementowano
```python
def task_edition():
    task_id = input("Podaj numer ID zadania do edycji: ")
    if task_id.isdecimal():
        task_id = int(task_id)